include/onnxruntime/core/framework/data_types.h

```diff
@@ -234,6 +239,7 @@ template <typename... Types>
 struct TensorContainedTypeSetter {
   static void SetTensorElementType(ONNX_NAMESPACE::TypeProto&);
   static void SetMapKeyType(ONNX_NAMESPACE::TypeProto&);
+  static int32_t GetElementType();
```

Comment:
>int32_t [](start = 9, length = 7)

Can we use enum to be more debugger friendly?

Reply:
We made a great effort to remove Protobuf constants from data_types.h so we can't bring it back.

---
In reply to: [342292647](https://github.com/microsoft/onnxruntime/pull/2307#discussion_r342292647) [](ancestors = 342292647)